inc/saiswitch.h

```diff
@@ -2873,6 +2879,32 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_HOSTIF_OPER_STATUS_UPDATE_MODE,
 
+    /**
+     * @brief VOQ chassis isolation setting.
+     *
+     * This attribute is for VOQ Chassis only
+     * This attribute is for configuring threshold value of active fabric ports below which VOQ Chassis isolation mechanism will trigger. Value of 0 means this attribute is not considered by switch.
```

Comment:
We need to explicitly call out that SAI adapter triggers callback when the number of UP fabric ports fall below the configured threshold and SAI adapter host callback processing should isolate by setting SAI_SWITCH_ATTR_SWITCH_ISOLATE to true.

This will use sai_switch_state_change_notification_fn as notification function? In that case that needs to be captured here as well?

Reply:
https://github.com/opencomputeproject/SAI/pull/1793/commits/64d77e084762eeb7ee35ac1ab1641b8584104c6d